content/docs/user-guide/interactivity/robotics/importing-robot.md

```diff
@@ -0,0 +1,111 @@
+
+URDF/XACRO files contain complete robot descriptions, including references to external geometry files. While primitive geometries can be defined directly within the URDF/XACRO file, it is common practice to use external mesh files in formats such as Collada DAE or STL to represent the visual and collision shapes of the robot.
+Robot models are typically made available in packages that include the URDF/XACRO file and additional files containing visualizations and collision shapes, whether those shapes are primitive geometries or external mesh files. These packages may be distributed as ROS workspaces, which makes them easier to use in ROS applications. ROS is a popular open-source framework for building robot software and includes many tools and libraries for working with URDF/XACRO files and robot models.
+
+ Thus typically robot models are made available in packages with additional files containing visualizations and collision shapes. Typical formats of mesh files in robotics are [Collada DAE](https://en.wikipedia.org/wiki/COLLADA) and [STL](https://en.wikipedia.org/wiki/STL_(file_format)). It is also a common practice to make available such robot packages as ROS workspaces, which make them easier to use in ROS applications.
+
+### URDF/XACRO importer - general description
+
+Both URDF and XACRO files can be imported into the O3DE using the importer included in the ROS2 Gem. The main features of the importer:
+
+- The importer can read both: URDF, as well as XACRO files.
+- The importer takes into account all features of the XACRO format, such as includes and variables.
+- Mesh files can be copied automatically to the asset folder of the O3DE project.
+- The importer creates a multi-body structure using articulations or classic rigid bodies and joints components.
+
+### Importing URDF/XACRO - step-by-step
+
+> Before you start:
+
+> 1. Importing URDF/XACRO files requires ROS-ready project with a ROS Gem enabled. Please refer to the [Project Configuration section](project-configuration.md) for details.
+> 2. In this section it is assumed, you have already created a O3DE project. Additional information on creating a robotic project can be found in the [Creating a Robotic Simulation section](creating-robotic-simulation.md)
+> 3. When using a XACRO files which are a part of a ROS workspace, it may be required to build the workspace first. Otherwise, the importer may fail to find all required files. Please refer to the documentation of the robot's description package for building instructions (typically you have to place it in the `src` folder of your ROS workspace and perform `colcon build`).
```

Comment:
```suggestion
When using robot definition files that are a part of a ROS package, you will often need to build a workspace with such package and its dependencies first. Otherwise, the importer may fail to find all required files. Please refer to the documentation of your robot description package, which is often included in its GitHub README file. Building typically involves creating a workspace and buidling it with `colcon build`.
```